server/lib/routes.js

```diff
@@ -81,13 +81,15 @@ module.exports = function (config, i18n) {
       '/signup_complete',
       '/confirm',
       '/settings',
-      '/settings/avatar',
       '/settings/avatar/change',
       '/settings/avatar/crop',
       '/settings/avatar/gravatar',
       '/settings/avatar/camera',
       '/settings/avatar/gravatar_permissions',
       '/settings/communication_preferences',
+      '/settings/delete_account',
```

Comment:
It looks like server tests need to be added for these new routes in `tests/server/routes.js`